snow/consensus/snowball/flat.go

```diff
@@ -10,17 +10,19 @@ import (
 
 var _ Consensus = (*Flat)(nil)
 
-func NewFlat(params Parameters, choice ids.ID) Consensus {
+func NewFlat(factory ConsensusFactory, params Parameters, choice ids.ID) Consensus {
+	sf := factory.New(params, choice)
```

Comment:
`sf` is typically shorthand for `snowflake` - should we just call this `nnary` or something? We could also just reduce the diff by removing the local variable and calling the function during the construction of the `Flat` struct like before

Reply:
Ya gonna move this into the struct creation